packages/anu-vue/src/components/select/ASelect.vue

```diff
@@ -107,7 +115,7 @@ function middleware() {
         ref="selectRef"
         readonly
         class="a-select-input cursor-pointer"
-        :value="modelValue.text ?? modelValue"
+        :value="selectedValueText"
```

Comment:
What if we change it to `modelValue ? (modelValue.text ?? modelValue) : undefined`

I guess this will probably work the same as the new changes but with less code

Reply:
Yep,it can be changed to this, I do this just for readability.

Reply:
Hey @IcetCode Can you please use `props.modelValue` instead of direct use of `modelValue` for future compatibility in template block? Anu defaults requires using `props.x` in template block.

Reply:
like this?
```ts
 :value="props.modelValue ? props.modelValue.text ?? props.modelValue : undefined"
```

Reply:
Yes 💯